doc/classes/InputEventMouseMotion.xml

```diff
@@ -12,6 +12,9 @@
 		<link title="3D Voxel Demo">https://godotengine.org/asset-library/asset/676</link>
 	</tutorials>
 	<members>
+		<member name="pen_inverted" type="bool" setter="set_pen_inverted" getter="get_pen_inverted" default="false">
+			Returns [code]true[/code] when using eraser-end of the pen.
```

Comment:
```suggestion
			Returns [code]true[/code] when using the eraser end of a stylus pen.
```
It's tangential to this PR but I'd also edit the other ones to change "of the pen" to "of a style pen", which makes it clearer that it's only applicable to cases where the event is emitted by a style pen (as we're in InputEventMouseMotion so the relationship is not immediately evident).